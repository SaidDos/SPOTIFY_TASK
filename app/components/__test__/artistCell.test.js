import 'react-native';
import React from 'react';
import ArtistCell from '../cells/artistCell';

import renderer from 'react-test-renderer';

it('should renders correctly', () => {
  const artist = {id: 1, name: 'said', type: 'human'};
  const tree = renderer.create(<ArtistCell artist={artist} />).toJSON();
  expect(tree).toMatchSnapshot();
});
