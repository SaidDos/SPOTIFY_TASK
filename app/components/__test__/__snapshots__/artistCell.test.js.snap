// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "lightgrey",
      "height": 70,
      "marginBottom": 20,
      "padding": 10,
      "width": 319.3,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 16,
        "fontWeight": "bold",
      }
    }
  >
    said
  </Text>
  <Text
    style={
      Object {
        "color": "grey",
      }
    }
  >
     
    human
  </Text>
</View>
`;
